cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2586,15 +2586,25 @@ Slice::Gen::DeclVisitor::visitClassDecl(const ClassDeclPtr& p)
 
     H << sp << nl << "class " << name << ';';
     H << nl << "using " << p->name() << "Ptr = ::Ice::SharedPtr<" << name << ">;";
+}
 
-    //
-    // _ice prefix because this function is in the Slice module namespace, where the user
-    // is allowed to define classes, functions etc. that start with _.
-    //
-    H << nl << "/// \\cond INTERNAL";
-    H << nl << _dllExport << "void _icePatchValuePtr(" << p->name() << "Ptr&, const "
-      << getUnqualified("::Ice::ValuePtr&", scope) << ");";
-    H << nl << "/// \\endcond";
+bool
+Slice::Gen::DeclVisitor::visitClassDefStart(const ClassDefPtr& p)
```

Comment:
Most of the changes in this file are _copying_ the Cpp11 visitors for classes to the Cpp98 with minimal changes.

At a later point--when the implementation of both mappings is closer--we should clean up this code and use the same visitors for both mappings.